src/Transfer.js

```diff
@@ -48,3 +48,11 @@ export default function Transfer (props) {
     </Grid.Column>
   );
 }
+
+export default function Transfer (props) {
+  const { api } = useSubstrate();
+  const result = (api.query.balances && api.tx.balances.transfer
+    ? <Main {...props} /> : null);
+  console.log(result);
```

Comment:
Do we really want this, or for your own debugging?

Reply:
That was for my own debugging. @shawntabrizi Fixed it in https://github.com/substrate-developer-hub/substrate-front-end-template/pull/59/